### ISSUE TITLE
Import data to SQLAlchemy

### ISSUE THREAD

**Issue by joelgoop** [demokratikollen/demokratikollen#3]
Import data from the rebuilt database into our own SQLAlchemy structure.

**Comment by joelgoop** [comment]
Members, groups, polls, votes, and chamber appointments work with procedure in README.md as of 42c1136.

Next steps:
- Documents (motioner, betänkanden). Requires some investigation of the raw data.
- Other appointments (utskott, etc.). Easy, but not too useful right now.

**Comment by jeinarsson** [comment]
Judging from
http://data.riksdagen.se/dokumentstatus/H101F%C3%B6U6.json
http://data.riksdagen.se/utskottsforslag/H101F%C3%B6U6.json
the information about which 'Punkt' in a 'Betänkande' a Poll refers to is stored in 'dokutskottsforslag'.

'Reservationer' is in 'dokmotforslag' with typ='reservation'.

Which 'betänkande' a 'motion' is treated in is hidden in 'dokforslag' field 'behandlas_i' which is a string with betänkande dokumentid eg '2013/14:FöU6', however information about 'Punkt' seems to be missing.

**Comment by rasmuse** [comment]
I started working a little bit on "betänkanden". Ran into a little bit of trouble in postgresifying and pouring into the database, most notably the ones reported here:
http://utveckling.riksdagen.se/2011/samlade-synpunkter-och-kommentarer-fran-beta-riksdagen-se/#comment-1775

I am well on my way to importing betänkanden 2010-2013 however, and will hopefully get this to work in the next few days.

**Comment by jeinarsson** [comment]
Good stuff. Please track your branch on GitHub too!
git push -u origin branch-name
http://stackoverflow.com/questions/2765421/push-a-new-local-branch-to-a-remote-git-repo-and-track-it-too

**Comment by rasmuse** [comment]
Check. 50064e9534aa6996da4c6ed7216bce4a1bc4aa23

**Comment by rasmuse** [comment]
OK, made some progress now. See 0994020.

However, it is now becoming rather clear that each .sql file from the Swedish parliament will probably need its own special treatment. For example, this file I added with "betänkande" documents had a few insert statements with duplicate columns. As we go along, the `postgresify` module will rather soon grow into a big pile of spaghetti and this is not something I can live with. Conclusion: we need to start making a better structure in the `postgresify` module. I already have a few ideas, so I might get to this quite soon.

Edit: I just opened #14 for this.
